101-sql-logical-server/azuredeploy.json

```diff
@@ -105,7 +105,8 @@
                   "name": "Default",
                   "apiVersion": "2018-06-01-preview",
                   "dependsOn": [
-                      "[resourceId('Microsoft.Sql/servers/', parameters('serverName'))]"
+                      "[resourceId('Microsoft.Sql/servers/', parameters('serverName'))]",
+                      "[concat('Microsoft.Sql/servers/', parameters('serverName'), '/securityAlertPolicies/Default')]"
```

Comment:
remove concat and use resourceId

resourceId('Microsoft.Sql/servers/securityAlertPolicies', parameters('serverName'), 'Default')

Reply:
dont know what happened but i lost my related forked repo. I had to submit this changes as a new PR.  https://github.com/Azure/azure-quickstart-templates/pull/6560 hope it is no trouble